Docs/source/glossary.rst

```diff
@@ -0,0 +1,308 @@
+**********
+Glossary
+**********
+
```

Comment:
I assume this is a copy-paste from BPS. It would be neat if we made a submodule for shared documentation, so we only have to maintain one copy of the glossary